css/main.css

```diff
@@ -0,0 +1,593 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&family=Inter:wght@400;700&display=swap');
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+html {
+  scroll-behavior: smooth;
+}
+
+a {
+  text-decoration: none;
+}
+
+hr {
+  width: 80vw;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 15px;
+  background-color: #3c3a39;
+  color: white;
+}
+
+#desktop-nav,
+#desktop-nav ul {
+  display: none;
+}
+
+#logo {
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 18px;
+  line-height: 24px;
+}
+
+.menu-bars {
+  background: transparent;
+  border: none;
+  outline: none;
+}
+
+.menu-bars i {
+  width: 18px;
+  height: 12px;
+  color: white;
+}
+
+#mobile-nav {
+  z-index: 9999;
+  position: absolute;
+  background-color: black;
+  width: 100%;
+  height: 100vh;
+  top: 0;
+  left: 0;
+  margin: 0;
+  padding: 0;
+  display: none;
+  padding-top: 50px;
+}
+
+#close-button-container {
+  width: 100%;
+  display: flex;
+  justify-content: flex-end;
+}
+
+#nav-close {
+  text-align: right;
+}
```

Comment:
Kindly give the close menu button a styling similar to the Figma because it's different there:

 - Your design:
   
![cl](https://user-images.githubusercontent.com/95412943/210756988-6e22c241-8e0e-4130-9809-0849672f6279.PNG)
- Figma's design:
 
![cl2](https://user-images.githubusercontent.com/95412943/210757135-99fe19c4-cdb1-4a53-8c29-ec664909ea49.PNG)